.github/workflows/ci.yml

```
@@ -167,3 +171,53 @@ jobs:
         fail_ci_if_error: false
         token: ${{ secrets.CODECOV_TOKEN }}
         verbose: true
+
+  # Performance and Benchmark Tests (Optional)
+  performance:
+    needs: [test]
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    # パフォーマンステストはmainブランチとPR時のみ実行
+    if: github.ref == 'refs/heads/main' || github.event_name == 'pull_request'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)